core/src/Cuda/Kokkos_Cuda_Team.hpp

```diff
@@ -130,28 +130,20 @@ class CudaTeamMember {
   KOKKOS_INLINE_FUNCTION int league_rank() const { return m_league_rank; }
   KOKKOS_INLINE_FUNCTION int league_size() const { return m_league_size; }
   KOKKOS_INLINE_FUNCTION int team_rank() const {
-#ifdef __CUDA_ARCH__
-    return threadIdx.y;
-#else
-    return 0;
-#endif
+    KOKKOS_IF_ON_DEVICE((return threadIdx.y;))
+    KOKKOS_IF_ON_DEVICE((return 0;))
```

Comment:
Maybe I'm misunderstanding something here, but if the previous behavior was to `return threadIdx.y` on device and `return 0` on host, shouldn't this be converted to
```c++
    KOKKOS_IF_ON_DEVICE((return threadIdx.y;))
    KOKKOS_IF_ON_HOST((return 0;))
```

Reply:
Fixed